src/GitHub.Api/Git/Tasks/GitAddTask.cs

```diff
@@ -22,6 +28,25 @@ class GitAddTask : ProcessTask<string>
             }
         }
 
+        public GitAddTask(AddFileOption addFileOption, CancellationToken token,
+            IOutputProcessor<string> processor = null) : base(token, processor ?? new SimpleOutputProcessor())
+        {
+            arguments = "add ";
+
+            switch (addFileOption)
+            {
+                case AddFileOption.All:
+                    arguments += "-A";
+                    break;
+
+                case AddFileOption.CurrentDirectory:
+                    arguments += ".";
```

Comment:
Both these options do the same thing since we're always running git operations from the root of the repository with full paths to all files, so might as well not have them?